drivers/bluetooth/hci/Kconfig

```diff
@@ -131,3 +131,19 @@ config BT_DRIVER_QUIRK_NO_AUTO_DLE
 
 	  This has to be enabled when the BLE controller connected is Zephyr
 	  open source controller.
+
+
+config BT_HCI_H4_SETUP
+	bool "Enable HCI vendor-specific Setup function"
+	depends on BT_H4
+	default n
+	help
+	  Enable the HCI vendor-specific Setup function.
+
+	  This option has to be enabled when the BT Controller requires execution
+	  of the vendor-specific commands sequence to initialize the BT Controller
+	  before the BT Host executes a Reset sequence.
+
+	  The user should generally avoid changing it via menuconfig or in 
+	  configuration files. This option are enabled by the vendor-specific
+	  HCI extension, where the Setup function is implemented.
```

Comment:
minor: GitHub is pointing out that there's no newline character at the end of this file.

Reply:
updated.